File: HYOWON_J/baekjoon/1541/1541.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim();

let calu = input.split('-');
let num = [];

for (let i of calu) {
    let sum = 0;
    let tmp = i.split('+');
    for (let j of tmp) {
        sum += parseInt(j);
    }
    num.push(sum);
}

let n = num[0];

for (let i = 1; i < num.length; i++) { 
    n -= num[i];
}

console.log(n);

File: HYOWON_J/baekjoon/1541/1541.py
import sys
sys.stdin = open("num.txt")

calu = input().split('-')
num = []

for i in calu:
    sum = 0
    tmp = i.split('+')
    for j in tmp:
        sum += int(j)
    num.append(sum)

n = num[0]

for i in range(1, len(num)):
    n -= num[i]

print(n)